.gitignore

```diff
@@ -1,9 +1,10 @@
 /.bundle/
-/.env
-/.env.test
+/.env.*.local
+/.env.local
```

Comment:
You should be able to do this with /.env*.local instead of the two lines

Reply:
done!